examples/terraform/packet/gobetween.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The KubeOne Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is mostly used in CI
```

Comment:
Can this be used by users as well? Will Terraform use it automatically? If yes, we should probably document it little bit.

Reply:
yes, given terraform configuration for packet will run this script on LB node automatically to provide "own LB" as packet lacks LB as a service functionality.

Reply:
Can you write in quickstart about that?